students/fahad/Lab 2/Part_1/Part_1.ino

```diff
@@ -0,0 +1,41 @@
+/**
+   Function: customPWM
+
+   Pin: Pin number to execute PWM on
+   dutyCycle: Percentage of brightness. 0 is off and 100 is maximum brightness
+   duration: how long the LED should be on for in microseconds
```

Comment:
Based on code below, duration seems to be in units of (number of pulses), not in microseconds.